samples/tetris/build.gradle

```diff
@@ -1,85 +1,114 @@
-apply plugin: 'org.jetbrains.kotlin.platform.native'
-
-def konanUserDir = System.getenv("KONAN_DATA_DIR") ?: "${System.getProperty("user.home")}/.konan"
-def resFile = file("$buildDir/konan/res/Tetris.res")
-
-components.main {
-    targets = ['macos_x64', 'linux_x64', 'raspberrypi', 'mingw_x64']
-    outputKinds = [EXECUTABLE]
-    baseName = 'tetris'
-
-    target 'macbook', {
-        linkerOpts "-F ${System.getProperty("user.home")}/Library/Frameworks -F /Library/Frameworks -framework SDL2"
```

Comment:
What's wrong with searching for a user framework?

Reply:
There are problems with running binary file which is built this way. Internally, it refers to SDL2 framework with relative path appended `@rpath` variable. When I run it from Mac OS X console this variable is magically resolves to the right disposition. When I run it from Java process (`exec`) `@rpath` is not resolved at all. This leads to run-time error in the application. Environment variables in Java process do not help (`LD_LIBRARY_PATH`, `DYLD_LIBRARY_PATH`).